### ISSUE TITLE
Default course - AA accessibility standards

### ISSUE THREAD

**Issue by jschof1** [adaptlearning/adapt_framework#3171]
### TODO
- [x] Aria labels for images
- [ ] Color contrast

**Comment by oliverfoster** [comment]
Could you be more specific. The background colour for which part?

Are you using your own theme or the standard theme [adapt-contrib-vanilla](https://github.com/adaptlearning/adapt-contrib-vanilla)?

Do you have any code references or screen shots? https://github.com/adaptlearning/adapt-contrib-vanilla/blob/master/less/_defaults/_colors.less#L3-L6

This is the colour you've given:
![image](https://user-images.githubusercontent.com/7974663/121785180-2fb4e900-cbb0-11eb-9225-a6cec1c86f17.png)
In Adapt we have these similar colours (item/popup, active item, drawer):
![image](https://user-images.githubusercontent.com/7974663/121785211-5bd06a00-cbb0-11eb-8d61-7dbb0cd882f6.png)
![image](https://user-images.githubusercontent.com/7974663/121785247-94704380-cbb0-11eb-90a9-cadb457bd050.png)

**Comment by oliverfoster** [comment]
As it transpires the vanilla course has some accessibility issues that we need to work out. I'm going to change the name of the issue to match this need and endeavour to sort those things out.

**Comment by jschof1** [comment]
Hey Oliver I have been very busy as of late so I have been unable to help
resolve the issues raised. I will have a look tomorrow.

**Comment by oliverfoster** [comment]
Don't worry. I had a run through whilst I was making a new extension specifically for visual accessibility [adapt-visua11y](https://github.com/cgkineo/adapt-visua11y). I see what you mean.

**Comment by oliverfoster** [comment]
need to change colours to be high contrast
